test/NuGet.Core.FuncTests/Dotnet.Integration.Test/DotnetRestoreTests.cs

```diff
@@ -2879,6 +2879,31 @@ await SimpleTestPackageUtility.CreatePackagesAsync(
             mockServer.Stop();
         }
 
+        [Theory]
+        [InlineData("../")]
```

Comment:
valid id but just longer than 100 chars should pass, so imo we should add a test for that.